lib/index.js

```diff
@@ -5,27 +5,28 @@ import Linter from './main'
 import Greeter from './greeter'
 import type { UI, Linter as LinterProvider } from './types'
 
+let greeter
```

Comment:
This change was required to get flow to report errors, it was previously ignoring all calls to `this.greeter` and `this.instance`, now it reports correctly